app/packs/src/apps/mydb/elements/details/ViewSpectra.js

```diff
@@ -351,36 +356,6 @@ class ViewSpectra extends React.Component {
     this.writeCommon(params, isMpy);
   }
 
-  // checkedToWrite(writing, predictions) {
```

Comment:
Could this section be removed instead of commented out?

Reply:
My bad, actually seeing now that this was already removed. I misread the diff (I'm not red-green blind though :sweat_smile:)